site/knock.go

```diff
@@ -163,38 +172,65 @@ func (knockEvent KnockEvent) doCountdown(wsConn *websocket.Conn, mqttClient mqtt
 	}
 	token := mqttClient.Publish("letmein2/timeout", 0, false, knockEvent.ShortLocation)
 	token.Wait()
-    knockEvent.Event = "TIMEOUT"
-    knockEvent.CurrentTime = 0;
+	knockEvent.Event = "TIMEOUT"
+	knockEvent.CurrentTime = 0
 	timeoutMessage, _ := json.Marshal(knockEvent)
+	bot.updateStatus(knockEvent.SlackMessageTS, "timeout", *knockEvent)
 	wsConn.WriteMessage(websocket.TextMessage, timeoutMessage)
 }
 
-func (knockEvent KnockEvent) readClientMsg(wsConn *websocket.Conn, mqttClient mqtt.Client, bot SlackBot) {
-    for {
-        _, message, err := wsConn.ReadMessage()
-        if err != nil {
-            log.Println("knockWatchForNvm:", err, ". exiting for ", knockEvent.ID)
-            return
-        }
-        clientMessageObject := KnockEvent{}
-        err = json.Unmarshal([]byte(message), &clientMessageObject)
-        fmt.Println("Recieved message from client in session ", knockEvent.ID, ". Message: ", clientMessageObject);
-        if err != nil {
-            log.Println("knockWatchForNvm:", err, ".")
-            return
-        }
-        if clientMessageObject.Event == "NEVERMIND" {
-            fmt.Println("Got NEVERMIND at ", clientMessageObject.Location)
-            wsConn.Close()
-            // TODO: support this on the device lol
-            token := mqttClient.Publish("letmein2/nvm", 0, false, clientMessageObject.Location)
-            token.Wait()
-        }
-        if clientMessageObject.Event == "NAME" {
-            fmt.Println("Got NAME: ", clientMessageObject.Name)
-            go bot.sendKnock(clientMessageObject.Name, location_map[clientMessageObject.Location])
-        }
-    }
```

Comment:
Please try to make VSCode not re-indent everything. Makes the diff hard to read. I think we're using 4 spaces for this project.

Update: [gofmt](https://go.dev/blog/gofmt) says use tabs. Can I ask you to re-indent with spaces _for now_ and then we can run `gofmt` in another PR?

TODO: Set up git CI to automatically gofmt or whatever

Reply:
I also gotta set up OKD hooks

Reply:
VS Code is currently set to use 4 spaces per tab, and the code is structured exactly like the original in my editor, looks like this is GitHub being weird. I'll try to see if changing it to just 4 spaces fixes it

Reply:
Converted all the tabs to 4 spaces each, should be solid now